### PR TITLE
Fix bad example in constructors.md

### PR DESCRIPTION
**Pull request by kshyatt**
cc @tkelman

### CODE REVIEW

**Review by KristofferC — APPROVED** [review]

File: doc/src/manual/constructors.md [hunk 1]
```diff
@@ -554,8 +554,8 @@ julia> type SummedArray{T<:Number,S<:Number}
            sum::S
        end
 
-julia> SummedArray(Int32[1; 2; 3], Int8(6))
-SummedArray{Int32,Int8}(Int32[1,2,3],6)
+julia> SummedArray(Int32[1; 2; 3], Int128(6))
+SummedArray{Int32,Int128}(Int32[1,2,3],6)
```
Comment (tkelman) [inline doc/src/manual/constructors.md]: the following paragraph says "For example, when `T` is `Int32`, we would like `S` to be `Int64`" though...

Comment (tkelman) [inline doc/src/manual/constructors.md]: maybe have this one give the `Int32` type that the following paragraph talks about preventing, then try to construct it in that block and show the error?

Comment (kshyatt) [inline doc/src/manual/constructors.md]: Sorry - you mean have the `Int128(6) -> Int32(6)` but then, in the next block, try the same thing again to show it doesn't work anymore?

Comment (tkelman) [inline doc/src/manual/constructors.md]: yeah

**Review by tkelman — APPROVED** [review]

**Review by tkelman — APPROVED** [review]
a working example at the end using the `SummedArray(a)` syntax and showing the result of the fancy constructor would be cool too, but guess we can leave that to later if needed

**Pull request merged**